### DISCUSSION TITLE
Good Prompt for Letter/Number Background

### DISCUSSION THREAD

**Discussion by yardsoul** [lllyasviel/Fooocus#2376] [Q&A] [answered]
Hi,

I am trying to generate image which contain background consist of something like trees or a cave channel that form into shape of letter or number. If anyone could help me come up with a decent prompt for this kind of work, I would be apricated. Thanks!

**Comment by ILikeToasters** [comment, marked as answer]
Not sure I understand exactly what you are looking to create but I think you will want to use a combination of text and image prompts. You will have to play around with the "stop at" and "weight". Just a couple of examples I did real quick with the letter M and number 3 (prompt was: looking out of a cave from inside): 
<img width="1031" alt="Screenshot 2024-02-27 192926" src="https://github.com/lllyasviel/Fooocus/assets/4357657/b22023ad-18f4-4d7f-8669-6e0935fded76">

<img width="371" alt="Screenshot 2024-02-27 192410" src="https://github.com/lllyasviel/Fooocus/assets/4357657/16c47a38-c355-44dc-a581-d6f50b76a854">

![2024-02-27_19-36-06_9464](https://github.com/lllyasviel/Fooocus/assets/4357657/a7e0364c-671e-4ba4-a12c-ad3bc18335c9)

**Reply by yardsoul** [reply]
This seem like what I am looking for. Thanks!
